legal-api/migrations/versions/01b28a2bb730_furnishings.py

```diff
@@ -0,0 +1,63 @@
+"""furnishings
+
+Revision ID: 01b28a2bb730
+Revises: 3083d361616e
+Create Date: 2024-06-12 20:10:44.723041
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '01b28a2bb730'
+down_revision = '3083d361616e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('furnishings',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('furnishing_type', sa.Enum('EMAIL', 'MAIL', 'GAZETTE', name='furnishing_type'), nullable=False),
+        sa.Column('furnishing_name', sa.Enum(
+            'DISSOLUTION_COMMENCEMENT_NO_AR',
+            'DISSOLUTION_COMMENCEMENT_NO_TR',
+            'DISSOLUTION_COMMENCEMENT_NO_AR_XPRO',
+            'DISSOLUTION_COMMENCEMENT_NO_TR_XPRO',
+            'INTENT_TO_DISSOLVE',
+            'INTENT_TO_DISSOLVE_XPRO',
+            'CORP_DISSOLVED',
+            'CORP_DISSOLVED_XPRO',
+            name='furnishing_name'
+            ),
+            nullable=False
+        ),
+        sa.Column('batch_id', sa.Integer(), nullable=False),
+        sa.Column('grouping_identifier', sa.Integer(), nullable=True),
```

Comment:
Can we add a sequence that we can use for this field when it's required?

Thinking something like this.  Can start at 1.

https://github.com/bcgov/lear/blob/c8572bcc07bcbe8c1be41b54dd080463a43ec96d/legal-api/migrations/versions/3d8eb786f4c2_add_coop_sequence.py#L19-L24